Tasks/PackerBuildV0/task.json

```diff
@@ -14,7 +14,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 208,
+        "Minor": 215,
```

Comment:
It should be 216 due to this [guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md):